gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/TextUtil.java

```diff
@@ -0,0 +1,27 @@
+package uk.nhs.adaptors.pss.translator.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class TextUtil {
+
+    public static String getLastLine(String text) {
+        if (text == null) {
+            return StringUtils.EMPTY;
+        }
+
+        if (!text.contains(StringUtils.LF)) {
+            return text;
+        }
+
+        String trimmed = text.trim();
+        return trimmed.substring(trimmed.lastIndexOf(StringUtils.LF)).trim();
+    }
+
+    public static String addLine(String text, String line) {
+        if (text == null) {
+            return StringUtils.EMPTY;
+        }
+
+        return text.concat(StringUtils.LF).concat(line);
```

Comment:
You could use ternary operator here:
```suggestion
        return text == null ? StringUtils.EMPTY : text.concat(StringUtils.LF).concat(line);
```